File: SDKs/Java/src/main/java/com/aspose/words/model/ProtectionData.java
package com.aspose.words.model;

public class ProtectionData {
  private String ProtectionType = null;
  /**
	 * getProtectionType
	 * Gets String
	 * @return ProtectionType
	 */
  public String getProtectionType() {
    return ProtectionType;
  }

	/**
	 * setProtectionType
	 * Sets String
	 * @param ProtectionType String
	 */
  public void setProtectionType(String ProtectionType) {
    this.ProtectionType = ProtectionType;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ProtectionData {\n");
    sb.append("  ProtectionType: ").append(ProtectionType).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/ProtectionData.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// 
  /// </summary>  
  public class ProtectionData 
  {                       
        /// <summary>
        /// Gets or sets ProtectionType
        /// </summary>    
        public string ProtectionType { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class ProtectionData {\n");
          sb.Append("  ProtectionType: ").Append(ProtectionType).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
